server/src/core/auth/services/token.service.ts

```diff
@@ -25,10 +25,9 @@ export class TokenService {
   ) {}
 
   async generateAccessToken(userId: string): Promise<TokenEntity> {
-    const expires = this.configService.get<string>('ACCESS_TOKEN_EXPIRES_IN');
-    assert(expires, '', InternalServerErrorException);
-    const expiresIn = ms(expires);
-    const expiresAt = addMilliseconds(new Date().getTime(), expiresIn);
+    const expiresIn = this.configService.get<string>('ACCESS_TOKEN_EXPIRES_IN');
+    assert(expiresIn, '', InternalServerErrorException);
+    const expiresAt = addMilliseconds(new Date().getTime(), ms(expiresIn));
```

Comment:
Could we specify the time unit in the variable name ? expiresInMs ? I often find it difficult to work with times because we're always facing seconds / milliseconds variables in the same codebase.